pipeline/cvp.groovy

```diff
@@ -112,14 +112,27 @@ node(nodeName) {
         def ciMessageMap = sharedLib.getCIMessageMap()
         def versions = sharedLib.fetchMajorMinorOSVersion('cvp')
         def releaseContent = sharedLib.readFromReleaseFile(
-            versions.major_version, versions.minor_version
+            versions.major_version, versions.minor_version, lockFlag=false
```

Comment:
you can remove the lockFlag=False parameter before merging if your testing is complete.

Reply:
done